docs/getting_started/authentication.md

```diff
@@ -0,0 +1,40 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Authentication
+
+Authentication mechanism is based on bearer token.
+How does it work?
+Both Jellyfish and Business Logic are configured with the same secret.
+This secret is used for authorizing HTTP requests (via HTTP authorization request header).
+When adding a new peer, Jellyfish creates a token for this peer and returns
+it to the Business Logic.
+Business Logic is then responsible for passing received token to the client that is willing
+to connect to Jellyfish and the client sends its token in the first message after opening
+Web Socket connection to Jellyfish.
+
+Token encodes peer id and room id. 
```

Comment:
I'd simplify that a bit. I'm not sure if we should include the last sentence of my suggestion - if there's another place describing that WS protocol, I'd keep such implementation detail out of this doc
```suggestion
When a new client wants to join a room, Business Logic (assuming the client is authorized to join) requests adding a new peer and receives a token generated by Jellyfish.
The token must then be passed to the client, that will use it to connect to the room as a new peer. The token encodes both peer and room IDs.
The token exchange is handled by our Client SDKs when connecting.
Under the hood, Jellyfish expects the token in the first message after opening the Web Socket connection.
```